app/models/skin.rb

```diff
@@ -353,10 +357,13 @@ def get_style_block(roles_to_include)
       if roles_to_include.include?(get_role)
         if self.filename.present?
           block += get_ie_comment(stylesheet_link(self.filename, get_media))
-        elsif self.css.present?
```

Comment:
We were kind of assuming that a skin could have CSS _or_ have wizard settings, but not both, and only including the custom CSS. Now what we do is add the wizard styles followed by the custom CSS you might've added to tweak the wizard settings.